README.md

```diff
@@ -57,9 +60,33 @@ unless real values don't help users know what to change.
 
 ```hcl
 
+module "watsonx_assistant" {
+  source            = "terraform-ibm-modules/watsonx-assistant/ibm"
+  region            = "us-south"
+  assistant_name    = "example-wx-assistant-basic"
```

Comment:
this is not correct. The variable you have is `watsonx_assistant_name`

Reply:
Done.